vulcan/data/_message.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from typing import AsyncIterator, List, Union
+
+from related import ChildField, IntegerField, SequenceField, StringField, immutable
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_MESSAGE
+from ..model import Attachment, DateTime, Serializable
+
+
+@immutable
+class Address(Serializable):
+    """An address.
+
+    :var str ~.login_id: login id
+    :var str ~.address_name: address name
+    :var str ~.address_hash: address hash
+    :var str ~.initials: initials
+    """
+
+    login_id: int = IntegerField(key="LoginId")
+    address_name: str = StringField(key="Address")
+    address_hash: str = StringField(key="AddressHash")
+    initials: str = StringField(key="Initials")
+
+
+@immutable
+class Message(Serializable):
+    """A message.
+
+    :var int ~.id: Message id
+    :var str ~.title: Subject of the message
+    :var `~vulcan.hebe.model.DateTime` ~.sent_datetime: Date with time when the message was sent
+    :var `~vulcan.hebe.model.DateTime` ~.read_datetime: Date with time when the message was read
+    :var int ~.status: Message status
```

Comment:
Does the status mean something? If it doesn't, is this field needed here?

Reply:
Probably it's something like `is read`

Reply:
Then maybe this needs an appropriate description. I don't know if I remember correctly, but wasn't the status something like the `folder` parameter you use for getting message list? (so 1 for received, 2 for sent, etc)

Reply:
Ok, I will check it and fix

Reply:
I checked it now, and it seems the status is always set to 1, no matter if the message is sent/received, read/unread, even deleted...